<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TcpIqPlotGUI</class>
 <widget class="RollupContents" name="TcpIqPlotGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TCP IQ Plot</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="IqPlotWidget" name="plotWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>200</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>TCP Port:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="port">
       <property name="minimum">
        <number>1024</number>
       </property>
       <property name="maximum">
        <number>65535</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="plotModeLabel">
       <property name="text">
        <string>Plot Mode:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="plotModeCombo">
       <item>
        <property name="text">
         <string>Time Domain (I/Q)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Constellation (I vs Q)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Magnitude</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Phase</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>FFT Spectrum</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="startButton">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stopButton">
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="persistenceCheckBox">
       <property name="text">
        <string>Persistence</string>
       </property>
       <property name="toolTip">
        <string>Keep old FFT data visible (for frequency scanning)</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="statusLabel">
     <property name="text">
      <string>Status: Idle</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RollupContents</class>
   <extends>QWidget</extends>
   <header>gui/rollupcontents.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>IqPlotWidget</class>
   <extends>QWidget</extends>
   <header>iqplotwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
